src/plugins/condition/components/inspector/ConditionalStylesView.vue

```diff
@@ -1,14 +1,43 @@
 <template>
-<div>Conditional Styles inspector view</div>
+<div class="c-properties__row--span-all">
+    <div class="controls">
+        <div class="preview">ABC</div>
+        <button class="c-icon-button--swatched icon-paint-bucket"
+                data-style-prop="backgroundColor"
+                @click="applyStyle"
+        ></button>
+        <button class="c-icon-button--swatched icon-line-horz"
+                data-style-prop="borderColor"
+                @click="applyStyle"
+        ></button>
+        <button class="c-icon-button--swatched icon-font"
+                data-style-prop="color"
+                @click="applyStyle"
+        ></button>
+    </div>
+</div>
 </template>
 
 <script>
 
 export default {
-    components: {
-    },
+    name: 'ConditionalStylesView',
     inject: [
         'openmct'
-    ]
+    ],
+    data() {
+        return {
+            backgroundColor: '#666',
+            borderColor: '#000',
+            color: '#ccc'
+        }
+    },
+    methods: {
+        applyStyle(ev) {
```

Comment:
It's visually difficult to tell which styles have been applied. Can you work with @charlesh88 to find out if there is active/inactive icon treatment we will have? If so we should add that in.

Reply:
It's in his UX diagram and I missed it. Done.

Reply:
There is a bug now in Display Layout palettes where the currently selected color is not being indicated, but should be. #2661.